hasura.planx.uk/metadata/tables.yaml

```diff
@@ -3095,19 +3095,28 @@
       using:
         foreign_key_constraint_on: flow_id
   insert_permissions:
+    - role: api
```

Comment:
Because we update `templated_flow_edits` via a Hasura event webhook, we're relying on `api` permissions here now rather than `teamEditor` or otherwise